keps/sig-node/4800-cpumanager-split-l3cache/README.md

```diff
@@ -0,0 +1,1161 @@
+should help to ensure that the tone and content of the `Summary` section is
+useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+## Motivation
+Our motivation is to reduce noisy neighbor scenarios that occur on systems with split L3 cache, which is available on both x86 and ARM architecture.
+The challenge with current kubelet’s cpu manager is that it is unaware of split L3 architecture and spreads CPU assignments across distributed L3s. This creates a noisy neighbor problem where multiple pods/containers are sharing the same L3 cache. In addition, pods spread against multiple L3 cache sees higher latency and reduced performance due to inter-cache access latency. For workload use cases that are sensitive to latency and are performance deterministic, minimizing the noisy neighbor condition in L3 can have significant improvements in performance.
+
+![Figure to the motivation](https://github.com/ajcaldelas/enhancements/blob/c7e423439608001cc837e2f5629dc87935353611/keps/sig-node/4800-cpumanager-split-l3cache/images/l3-alignment-motivation.png)
+
+The figure below highlights performance gain when pod placement is aligned to an L3 group. In this exampleTPCC/My-SQL  is the workload deployed. There was a 18% uplift compared to default behavior.  Other workloads might see  higher gains.  For Stream workload, highlighted in the original design doc, the performance uplift was approximately 20%.
+
+![Figure to the results of l3 cache alignment](https://github.com/ajcaldelas/enhancements/blob/c7e423439608001cc837e2f5629dc87935353611/keps/sig-node/4800-cpumanager-split-l3cache/images/results.png) ![Figure of results](https://github.com/ajcaldelas/enhancements/blob/c7e423439608001cc837e2f5629dc87935353611/keps/sig-node/4800-cpumanager-split-l3cache/images/chart.png)
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+### Goals
+
+- Introduce a new CPU Manager policy option that assigns CPU within the same grouping of uncorecache or L3 cache to pods and container scope.  
+- Minimize the number of CPUs assignments from different uncorecache/L3 grouping to pods and container scope.
+- Cross-die could also decrease performance.  Add support for multiple socket systems.
```

Comment:
Forgive me I don't really understand the third goal. Is Cross-die could also decrease performance a goal? Or adding support for multiple socket systems?

Reply:
updated goal for more clarity